packages/core/src/browser/widgets/react-renderer.tsx

```diff
@@ -19,6 +19,9 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { Disposable } from '../../common';
 
+export type RendererHost = HTMLElement;
```

Comment:
Looks like these declarations might make more sense in the `breadcrumb-popup-container` file?